### PR TITLE
@ #511 | should update getting started docs

### PR DESCRIPTION
**Pull request by hieptranquoc**
connect #511 
please check this PR, thank you

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -54,13 +54,13 @@ Open the terminal window:
 
 2. Install ``virtualbox`` and ``vagrant``
 
-   - Install ``virtualbox``:
+   - Install ``virtualbox``: ( require version: >= 5.2 )
```
Comment (datphan) [inline docs/getting_started.rst]: the `()` should place before the `:`

Install ``virtualbox`` (v5.2 or above):

File: docs/getting_started.rst [hunk 2]
```diff
@@ -19,8 +19,8 @@ Please follow the installation instruction below, after that, the instruction st
 
     - We recommend the following tested and supported platforms:
 
-      + macOS Sierra
-      + Ubuntu 16.04
+      + macOS Sierra and above
```
Comment (datphan) [inline docs/getting_started.rst:19]: `or` above

File: docs/getting_started.rst [hunk 3]
```diff
@@ -19,8 +19,8 @@ Please follow the installation instruction below, after that, the instruction st
...
+      + Ubuntu 16.04 and above
       + Windows 10, Windows 8, Windows 7
```
Comment (phuonglm) [inline docs/getting_started.rst]: windows 7 SP1 is better,
need 64 bit version.

File: docs/getting_started.rst [hunk 4]
```diff
@@ -279,9 +279,8 @@ teracy-dev Git Clone and Vagrant Up
    ..  code-block:: bash
 
        $ cd ~/
-       $ git clone https://github.com/teracyhq/dev.git teracy-dev
+       $ git clone https://github.com/teracyhq/dev.git -b v0.6.0-a5 teracy-dev
        $ cd teracy-dev
-       $ git checkout develop
```
Comment (datphan) [inline docs/getting_started.rst]: shoud have `$ git checkout v0.6.0-a5` for visual impact, like I usually don't like to use `git clone url -b`

**Comment by hieptranquoc** [comment]
I've updated, please check again! @datphan

File: docs/getting_started.rst [hunk 5]
```diff
@@ -280,8 +279,8 @@ teracy-dev Git Clone and Vagrant Up
 
        $ cd ~/
        $ git clone https://github.com/teracyhq/dev.git teracy-dev
+       $ git checkout v0.6.0-a5
```
Comment (datphan) [inline docs/getting_started.rst]: We have to cd repo first before checkout branch

Comment (hieptranquoc) [reply docs/getting_started.rst]: @datphan I've updated! Please check again! Thank you

**Review by datphan — DISMISSED** [review]

**Review by hoavt — DISMISSED** [review]

File: docs/getting_started.rst [hunk 6]
```diff
@@ -281,7 +280,7 @@ teracy-dev Git Clone and Vagrant Up
        $ cd ~/
        $ git clone https://github.com/teracyhq/dev.git teracy-dev
        $ cd teracy-dev
-       $ git checkout develop
+       $ git checkout v0.6.0-a5
```
Comment (hoatle) [inline docs/getting_started.rst]: this will not work, just check out the develop branch so that others can follow

**Review by hoatle — CHANGES_REQUESTED** [review]
make sure to check and upgrade on all platforms and specify the fixed latest stable working versions of packages.

**Comment by hieptranquoc** [comment]
I've updated! Please check again! Thank you.

File: docs/getting_started.rst (hunk 3 above)
Comment (phuonglm) [reply docs/getting_started.rst]: Please note about power shell version

**Review by phuonglm — CHANGES_REQUESTED** [review]

**Comment by hieptranquoc** [comment]
@hoatle I've updated! Please check this PR! Thank you!

**Comment by hoatle** [comment]
need @hoavt to review

**Review by hoavt — APPROVED** [review]

**Pull request merged**